src/webview/openshift-terminal/openShiftTerminal.ts

```diff
@@ -522,23 +522,23 @@ export class OpenShiftTerminalManager implements WebviewViewProvider {
         // wait until the webview is ready to receive requests to create terminals
         await this.webviewResolved;
 
-        const [cmd, ...args] = `${commandText}`.split(' ');
+        const tool = `${commandText}`.split(' ')[0];
```

Comment:
CRC setup failing for the mentioned path `C:\\Program Files\\Red Hat OpenShift Local\\crc.exe setup`

Reply:
It should work now if the path has spaces. I don't have access to a Windows computer currently, so please let me know if it works for you.